### PR TITLE
Improve discussion notification email UI

### PR DESCRIPTION
**Pull request by hrvojevu**
As a part 2 of issue https://github.com/ExtensionEngine/tailor/issues/517, this PR:
- removes activity url from discussion notification email
- displays last 3 discussion messages in addition to the last one
- updates discussion email styling
- updates email `renderHtml` function since `mjml2html` messes up `mustache` `{{#}}, {{/}}, {{^}}` tags

Screen Shot 2020-08-31 at 11 10 08 AM

### Note:
Base branch must be changed to `develop` after https://github.com/ExtensionEngine/tailor/pull/586 has been merged in

### CODE REVIEW

**Comment by kjuej** [comment]
@hrvojevu It seems if there's less than 5 comments in total, only the last comment will be shown in the notification email. Is this intended?
Screen Shot 2020-09-01 at 11 44 12

**Comment by kjuej** [comment]
@hrvojevu @underscope @vladimyr Seems like the issue above only occurs on gmail. Also the logo is missing on gmail as well.

**Comment by hrvojevu** [comment]
This is the link that gmail produces:

`https://ci4.googleusercontent.com/proxy/mSnVz8PRZQyuYaDvkJpLIHXatki_PiJia-qK2f8d9Q-OxgF1V7kp4P9-KD9enBmliJXhgiKa0XWZAXlS053N1NIa2i5_-C9UxRnxVw=s0-d-e1-ft#http://127.0.0.1:3000/assets/img/default-logo-compact.svg`

I suppose this doesn't work due to localhost and would work in production environment.

File: server/shared/mail/templates/components/head.mjml [hunk 1]
```diff
@@ -20,10 +20,12 @@
       font-size="12px"
       background-color="white" />
     <mj-class name="divider"
-      padding="20px 25px"
+      padding="20px 10px"
       border-width="1px"
       border-color="#d2d2d2"
       border-style="dashed" />
+    <mj-class name="avatar"
+      padding="10px" />
```
Comment (underscope) [inline server/shared/mail/templates/components/head.mjml]: Consider inlining (only 2 attrs)

File: server/comment/hooks.js [hunk 2]
```diff
@@ -39,13 +39,19 @@ async function sendEmailNotification(comment, db) {
     ]
   });
   const { author, repository, activity } = comment;
+  const lastFourComments = await activity.getComments({
```
Comment (underscope) [inline server/comment/hooks.js]: Define this more generic like bellow (`previousComments`). Use offset to eliminate 1.

File: server/shared/mail/templates/comment.mjml [hunk 3]
```diff
@@ -3,33 +3,52 @@
...
+            </td>
+            <td style="padding: 0 0 0 20px">
+              <div class="label">{{author.label}}</div>
+              <div>{{content}}</div>
+            </td>
+          </tr>
+        </mj-table>
+       </mj-column>
+    </mj-section>
+    {{#previousComments}}
```
Comment (underscope) [inline server/shared/mail/templates/comment.mjml:23]: Consider extracting into separate component.

File: server/shared/mail/templates/comment.mjml [hunk 4]
```diff
@@ -3,33 +3,52 @@
...
-          font-size="16px">
-          {{content}}
+          <b>{{author.label}}</b> left a comment on <b>{{repositoryName}}</b>, <b>{{topic}}</b>.
         </mj-text>
       </mj-column>
     </mj-section>
     <mj-section padding="0 10px">
-      <mj-column border-top="1px solid #ccc">
+			<mj-column border-bottom="2px dashed #d2d2d2">
+        <mj-table padding="10px">
```
Comment (underscope) [inline server/shared/mail/templates/comment.mjml]: See if there is a better way of implementing this layout

File: server/shared/mail/templates/comment.mjml (hunk 3 above)
Comment (hrvojevu) [reply server/shared/mail/templates/comment.mjml:23]: I don't think it is possible due to problems described in the render function. If we were to import separate component here, `mustache` render would not catch that component and `mjml2html ` would mess up mustache tags, second mustache render would not be able to render that component properly.

File: server/shared/mail/templates/comment.mjml [hunk 5]
```diff
@@ -3,33 +3,52 @@
...
+          <tr class="content" style="display: flex;">
+            <td>
+              <img class="avatar" src="{{author.imgUrl}}" />
```
Comment (underscope) [inline server/shared/mail/templates/comment.mjml]: Data URLs (if image is set) won't work in Gmail (https://stackoverflow.com/questions/8580355/can-you-send-images-in-datauri-format-to-gmail)

Comment (hrvojevu) [reply server/shared/mail/templates/comment.mjml]: Data URLs are not used in this instance, but a URL pointing to localhost. I figure this should work when these images are hosted on publicly available domain.

File: server/shared/mail/templates/comment.mjml (hunk 4 above)
Comment (hrvojevu) [reply server/shared/mail/templates/comment.mjml]: Initially I have tried using the common mjml tags (mj-section, mj-column) but afaik it is not possible to achieve this exact layout.

**Pull request merged**